src/Quasar/Mount/MongoDB.purs

```diff
@@ -54,59 +49,52 @@ instance showAuth ∷ Show Auth where
     "(Auth { path: " <> show path <> ", credentials: " <> show credentials <> " })"
 
 type Config =
-  { hosts ∷ NonEmpty Array Host
+  { hosts ∷ URI.QURIHost
```

Comment:
This is incorrect now; mongo URIs will need a separate options setup / type configuration, since they're allowed to have multiple hosts. The `Data.URI.Extra.MultiHostPortPair` is for this.

Reply:
it's. `QURIHost`
`type QURIHost = URI.MultiHostPortPair URI.Host URI.Port`

Reply:
Oh I see, it's the opposite problem then - every other URI should not support multiple hosts :smile: (it only did that before because we couldn't express otherwise with the types).